frontend/src/routes/settings/content/+page.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+    import type { PageData } from './$types';
+    
+    export let data: PageData;
+</script>
+
+<p>WIP</p>
+<img src="https://c.tenor.com/x8v1oNUOmg4AAAAd/tenor.gif" alt="WIP" class="w-1/2 mx-auto">
```

Comment:
Lets remove this before pushing to main. lol

Reply:
Shh, this is needed, soon there will be loaders with your face......